### ISSUE TITLE
problems with modernizr

### ISSUE THREAD

**Issue by introvert** [yeoman/generator-webapp#605]
Hello,

I'm experiencing problems with modernizr when the library is built using the webapp approach.

When I deploy the site, I'm unable to scroll the page and I get the following error:
`modernizr.a9ab48a0.js:1 Uncaught TypeError: Cannot read property 'sheet' of null`

It looks like something breaks the library after it gets built. 

Any idea what this happens? It started appearing when I started using `gulp-rev-all`

It also seems that webapp is using outdated approaches with using the modernizer (the version included is 2.8.3). Any future plans for caregiving into this?

**Comment by silvenon** [comment]
It sounds like you made some modifications to your setup since you initialized it. Could you provide steps to reproduce this?

**Comment by mtompkins** [comment]
If I might share my Modernizr approach after a few days of struggling with it.

Update to a 3.x version with `bower`
Create a configuration using the online download page: https://modernizr.com/download
Build and download a command line config `modernizr-config.json`

Add a gulp task, modifying location of `modernizr-config.json` as necessary:

```
// Modernizr
var spawn = require('child_process').spawn;
gulp.task('modernizr', function (cb) {
  var cmd = spawn('bower_components/modernizr/bin/modernizr', [
        '-c', 'modernizr-config.json', 
        '-d', 'app/scripts/modernizr-custom.js',
        '-u', '-q'
  ], {stdio: 'inherit'});
  cmd.on('close', function (code) {
      console.log('Modernizr exit code: ' + code);
      cb(code);
  });
});
```
Change index.html page to reflect changes:
```
<!-- build:js scripts/vendor/modernizr.js -->
<script type="text/javascript" src='scripts/modernizr-custom.js'></script>
<-- endbuild -->
```

**Comment by UlisesGascon** [comment]
I assume that this bug was fixed in #737. If not we can re-open it again 😃
